sycl/test-e2e/DeviceLib/string_test.cpp

```diff
@@ -1,10 +1,11 @@
 // UNSUPPORTED: hip
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -fno-builtin %s -o %t.out
+// RUN: %{build} -fno-builtin -o %t.out
+// RUN: %{run} %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
```suggestion
```

Reply:
Same as above - I need an extra feature to change lines 9-10.